components/speedreader/rust/ffi/Cargo.lock

```diff
@@ -280,9 +280,9 @@ dependencies = [
 
 [[package]]
 name = "flate2"
-version = "1.0.19"
```

Comment:
@mihaiplesa do you know why this wasn't included in #7871?

Reply:
Renovate does one at a time (at least for Rust). There might be an option to change. Also, if it needs CI let me know or feel free to take it out of draft.

Reply:
I think it should be okay to merge, just that if these PRs are leaving behind un-updated lockfiles they'll require manual cleanup PRs afterwards